File: calc/python/ini/apigear_calc/calculator.py
from apigear_calc.api import api

class Calculator(api.ICalculator):
    def __init__(self):
        self.__total : int = 0

    @property
    def total(self):
        return self.__total

    def add(self, a: int) -> None:
        return None

    def substract(self, a: int) -> None:
        return None

    def multiply(self, a: int) -> None:
        return None

    def divide(self, a: int) -> None:
        return None

    def clear(self) -> None:
        return None

File: calc/golang/ini/apigear/calc/calculator.go
package calc

import "project/apigear/calc/api"

type calculator struct {
}

func NewCalculator() api.Calculator {
  obj := calculator{}
  return &obj
}

func (c *calculator) GetTotal() int {
  return 0
}

func (c *calculator) SetTotal(total int) {  
}

func (c *calculator) Add(a int)  {
  return 
}

func (c *calculator) Substract(a int)  {
  return 
}

func (c *calculator) Multiply(a int)  {
  return 
}

func (c *calculator) Divide(a int)  {
  return 
}

func (c *calculator) Clear()  {
  return 
}
